bkrdoc/analysis/generator/documentation_generator.py

```diff
@@ -236,7 +236,39 @@ def get_overall_credibility(self):
             if not self.is_phase_outside(phase):
                 unknown_commands += phase.get_unknown_commands()
                 total_commands += phase.get_total_commands()
-        return DocumentationCredibility(unknown_commands, total_commands).get_credibility()
+        return credibility.DocumentationCredibility(unknown_commands, total_commands).get_credibility()
+
+    def generate_phase_documentation(self, phase):
+        """
+        Transforms DocumentationInformation into small classes using GetInformation
+        """
+        information_translator = get_information.GetInformation()
+        for information in phase.documentation_units:
+            if information:
+                phase.phase_documentation_information.append(information_translator.get_information_from_facts(information))
+
+    def print_phase_documentation(self, phase, cmd_options):
+        """
+        Prints nature language information
+        :param cmd_options: possible command line options
+        """
+        self.print_phase_name_with_documentation_credibility(phase)
+        conditions = conditions_for_commands.ConditionsForCommands()
+
+        for information in phase.phase_documentation_information:
+            if cmd_options.log_in or cmd_options.print_all:
+                information.print_information()
+            elif not conditions.is_rllog_command(information.get_command_name()):
+                information.print_information()
+
+    def print_phase_name_with_documentation_credibility(self, phase):
+        inf = phase.phase_name + " [Unknown commands: " + str(phase.get_unknown_commands()) \
+                              + ", Total: " + str(phase.get_total_commands()) \
+                              + ", Documentation credibility: " + self.get_phase_credibility(phase).get_credibility() + "]"
+        print(inf)
```

Comment:
For easy string formatting you can use format. Above case in format.
inf  = "{0} [Unknown commands: {1}, Total: {2}, Documentation credibility: {3}]".format(phase.phase_name, phase.get_unknown_commands(), phase.get_total_commands(), self.get_phase_credibility(phase).get_credibility())

I'm using format cause is easy to format string with it and you will not make mistakes with missing/more white spaces.
You can read about format here: https://docs.python.org/2/library/string.html#format-string-syntax